lib/src/signed_video_h26x_common.c

```diff
@@ -977,7 +974,7 @@ hash_and_copy_to_ref(signed_video_t *self, const h26x_nalu_t *nalu, uint8_t *has
     // Copy the |nalu_hash| to |reference_hash| to be used in hash_with_reference().
     memcpy(reference_hash, hash, hash_size);
     // Copy the |hash| to |linked_hash| while linked hash prinsiple is used.
-    if (self->linked_hash_on) get_linked_hash(self, hash, hash_size);
+    if (self->linked_hash_on) set_linked_hash(self, hash, hash_size);
```

Comment:
Can't you always do this? The only difference is that when `linked_hash_on = false` the stored linked hashes will never be used. If you always do this operation you will execute more of the code which is better for testing and valgrind.